src/constants/config.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  BASE_URL:
+    'https://cors-anywhere.herokuapp.com/https://api.pushshift.io/reddit/',
```

Comment:
Why do you use `https://cors-anywhere.herokuapp.com` and not directly `https://api.pushshift.io/reddit/`

Reply:
First time I tried API it gives me cors origin error, now everything works without cors-anywhere